packages-web/maps/src/Maps.tsx

```diff
@@ -0,0 +1,60 @@
+import { hot } from "react-hot-loader/root";
+import { createElement, ReactNode, useEffect, useState } from "react";
+import { MapsContainerProps, Marker } from "../typings";
+import { ValueStatus } from "mendix";
+
+import { getCurrentUserLocation, translateZoom, useLocationResolver } from "./utils";
+import { MapSwitcher } from "./components/MapSwitcher";
+
+import "leaflet/dist/leaflet.css";
+import "./ui/Maps.css";
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.webpack.css";
+import "leaflet-defaulticon-compatibility";
+
+const Maps = (props: MapsContainerProps): ReactNode => {
+    const [, locations] = useLocationResolver(
```

Comment:
const [, ] ?

Reply:
Supressing first argument or the future proof hook :)